lahja/endpoint.py

```diff
@@ -69,6 +68,10 @@ class Broadcast(NamedTuple):
     config: Optional[BroadcastConfig]
 
 
+class ReaderAtEof(Exception):
```

Comment:
Maybe that should also move to `exceptions.py` and inherit from `LahjaException`?